### ISSUE TITLE
nodejs-models example throws ValueError: Unknown layer: MultiplyBeta.

### ISSUE THREAD

**Issue by Thisen** [thekevinscott/UpscalerJS#890]
Running the `nodejs-models` example locally on Node 18 throws the following error:
```sh
/UpscalerJS/examples/nodejs-model/node_modules/@tensorflow/tfjs-layers/dist/tf-layers.node.js:276
        var _this = _super.call(this, message) || this;
                           ^

ValueError: Unknown layer: MultiplyBeta. This may be due to one of the following reasons:
1. The layer is defined in Python, in which case it needs to be ported to TensorFlow.js or your JavaScript code.
2. The custom layer is defined in JavaScript, but is not registered properly with tf.serialization.registerClass().
```

**Comment by thekevinscott** [comment]
Thanks for the report. I've published a new version, `1.0.0-beta.11`, that should resolve this issue.

[Check it out in Code Sandbox](https://githubbox.com/thekevinscott/upscalerjs/tree/main/examples/nodejs-model) (try forking it for their new editor, or [use this link directly](https://codesandbox.io/p/sandbox/thirsty-flower-nv2v97?file=%2Fsrc%2Findex.js)).

**Comment by Thisen** [comment]
Works. Thanks!
